src/test/java/api/loans/LoanAPITests.java

```diff
@@ -1582,6 +1583,27 @@ public void canCreateMultipleLoansWithServicePoints()
 
   }
 
+  @Test
+  public void SingleLoanContainBorrowerInformation()
+      throws InterruptedException, MalformedURLException, TimeoutException, ExecutionException {
+    IndividualResource steve = usersFixture.steve();
+    InventoryItemResource inventoryItemResource = itemsFixture.basedUponDunkirk();
+    IndividualResource individualResource = loansFixture.checkOutByBarcode(inventoryItemResource, steve);
+    Response loan = loansClient.getById(individualResource.getId());
+    loanHasExpectedProperties(loan.getJson());
+  }
+
+  @Test
+  public void MultipleLoansContainBorrowerInformation()
```

Comment:
Why has this test been introduced, as the existing `canPageLoans`, `canSearchByUserId`, `canFilterByLoanStatus` already checks these properties?

Reply:
I wanted to explicitely check get single and get many loans, as the borrower info was added in those two places. I also agree the existing tests should be reused for this check.

Reply:
Tests have been re-arranged in the last commit